showpass/Showpass-E2E-tests
# TA-100 Verify event restriction items appearance

**Description:**
This user story will be verifying that once ‘Important info & restrictions’ are added to an event on the event configuration page, these restrictions show up on the front-end side.
1. Make sure all the ‘Important Info & Restrictions’ items are added to an event on the event configuration portal 
2. Open the event on the front-end portal
3. Make sure all the restriction items show up and are correct

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-100

**How to test:**
Run the following spec and ensure all tests pass: cypress/e2e/events/verify-events.cy.js

**Test execution results:**

[Cypress Cloud test results](https://cloud.cypress.io/projects/e9axi7/runs/181/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D)
